lib/tasks/check_syntax.rake

```diff
@@ -34,7 +34,8 @@ namespace :check do
       end
 
       res = `ruby -c -w #{file}`
-      puts "#{file}: #{res}" if verbose
+      res = res.lines.reject { |s| s == "Syntax OK\n" }.join ""
```

Comment:
it is a bit problematic. We do not force `LANG=C` above So this way it can show localized message which do not match. If we use `LANG=C` it means that developers won't see localized error message. Which in the end is probably fine. So please add it above.